apps/homescreen-ng/README.md

```diff
@@ -0,0 +1,30 @@
+# homescreen-ng
+
+homescreen-ng is a prototype homescreen replacement for FirefoxOS, that leverages web components to help improve reusability and enforce code separation. Performance, UX and maintainability are its top priorities.
+
+## Requirements
+
+homescreen-ng requires a version of FirefoxOS >= 2.3, and that web components are enabled. This can be done by enabling the device preference `dom.webcomponents.enabled`.
+
+## Installing and running
+
+First, make sure submodules are updated by running:
+
+```
+git submodule init
```

Comment:
I am pretty sure that this is not going to work for our build process. Do we still need submodules here?